### PR TITLE
Add Dropdown Icon

### LINKED ISSUE #3324: Add dropdown icon adjacent to the navbar items where required
**We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

We used to have buttons in the Navbar some time back which clearly guide people to explore options but with links in the Navbar, a site visitor may think of them as a direct link. So, dropdown icon needs to be added adjacent to the dropdown Navbar link.

Hint: You need to make changes in this https://github.com/publiclab/plots2/blob/master/app/views/layouts/_header.html.erb file.

dropdown_gci

### PR DESCRIPTION
**Pull request by jmliu1417**
Fixes #3324  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by gauravano** [comment]
Hi @jmliu1417, please add the screenshot/GIF of the updated view. Also, make sure to check the color of the icon as @Paarmita  mentioned. Thanks!

**Comment by DakshGondaliya** [comment]
@jmliu1417 Remove **" f "** from the line  no. 94 in your PR

**Comment by HarshKhandeparkar** [comment]
Ok got it.

**Comment by jmliu1417** [comment]
@gauravano I added the color element in https://github.com/jmliu1417/plots2/blob/master/app/assets/stylesheets/style.css
Also, I'm still having issues with running plots2. Is there a way I can submit this project without need to run the code?

**Comment by gauravano** [comment]
Hi @jmliu1417 I will run your PR on my system for now, but please install plots2 for further tasks. At which part you are stuck during installation?

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -30,7 +30,7 @@
       <ul class="nav navbar-nav navbar-left">
         <li class="dropdown"><i class="fas fa-caret-down"></i>
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
        <li class="dropdown">
```

**Comment by gauravano** [comment]
And, for styling add this code  

```
a .fa {
 color: #bcb2b2;
}
```

-  at  https://github.com/publiclab/plots2/blob/master/app/assets/stylesheets/dashboard.css#L7

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -30,7 +30,7 @@
...
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-            Get Involved
+            Get Involved  <i class="fa fa-caret-down"></i>
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
            Get Involved  <i class="fa fa-white fa-caret-down"></i>
```

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -51,7 +51,7 @@
 
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-            <%= t('layout._header.about.about_title') %>
+            <%= t('layout._header.about.about_title') %> <i class="fa fa-caret-down"></i>
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
            <%= t('layout._header.about.about_title') %> <i class="fa fa-white fa-caret-down"></i>
```

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -91,7 +91,7 @@
           <% unless params[:action] == "register" || params[:action] == "signup" %>
             <% if current_user %>
               <a class="dropdown-toggle" data-toggle="dropdown" href="#" style="color: white">
-                Profile
+                Profile  <i class="fa fa-caret-down"></i>
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
                Profile  <i class="fa fa-white fa-caret-down"></i>
```

**Comment by Eshaabhasin** [comment]
everything is correct

**Comment by jmliu1417** [comment]
@gauravano Can I get more feedback on improvements please?

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -50,7 +53,9 @@
 
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-            <%= t('layout._header.about.about_title') %> <i class="fa fa-caret-down" style="color:grey"></i>
+
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
```

File: app/views/layouts/_header.html.erb [hunk 6]
```diff
@@ -30,7 +30,10 @@
       <ul class="nav navbar-nav navbar-left">
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
+
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
```

File: app/views/layouts/_header.html.erb [hunk 7]
```diff
@@ -30,7 +30,10 @@
...
+
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
```

File: app/views/layouts/_header.html.erb [hunk 8]
```diff
@@ -50,7 +53,9 @@
 
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-            <%= t('layout._header.about.about_title') %> <i class="fa fa-caret-down" style="color:grey"></i>
+
+            <%= t('layout._header.about.about_title') %> <i class="fa fa-white fa-caret-down"></i>
+
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
```

**Comment by jmliu1417** [comment]
@gauravano is this task ready for submission

File: app/views/layouts/_header.html.erb [hunk 9]
```diff
@@ -31,6 +31,7 @@
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
             Get Involved <i class="fa fa-caret-down" style="color:grey"></i>
+
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
```

File: app/views/layouts/_header.html.erb [hunk 10]
```diff
@@ -31,6 +31,7 @@
         <li class="dropdown">
           <a class="dropdown-toggle" data-toggle="dropdown" href="#">
             Get Involved <i class="fa fa-caret-down" style="color:grey"></i>
```
Comment (gauravano) [inline app/views/layouts/_header.html.erb]:
```suggestion
            Get Involved <i class="fa fa-white fa-caret-down"></i>
```

**Comment by gauravano** [comment]
@jywarren here's the screenshot

dropdown_gci_2

P.S: Let me know if caret color need changes (I used white as caret are small and grey was kind of making them invisible)

**Comment by jywarren** [comment]
Amazing! Fantastic, thanks!!!

**Pull request merged**